### ISSUE TITLE
Fix PSP to use updated names for macros and symbols

### ISSUE THREAD

**Issue by skliper** [nasa/PSP#84]
CFE 6.6 addresses a number of naming convention inconsistencies, and many of the symbol names are updated to be clearer about the intent and purpose of the symbol.

As part of CFE 6.6 a backward-compatibility mapping is also provided.

This ticket is to do the corresponding updates to the PSP to use the correct new-style name, and not rely on the backward compatibility macro anymore.  This will need to be done before the compatibility names can be turned off.

**Comment by skliper** [comment]
_Imported from trac issue 80.  Created by jphickey on 2017-09-26T11:12:55, last modified: 2019-07-11T14:34:01_

**Comment by skliper** [comment]
_Trac comment by jphickey on 2017-09-26 11:38:44:_

Proposed change:

Commit [changeset:224e54f] branch `trac-80-psp-use-new-symbol-names`

**Comment by skliper** [comment]
_Trac comment by glimes on 2017-10-06 12:38:22:_

The change set looks good by eye, but I want to see a clean compile; the merge
that included this change had build errors and I do not see a newer build that
succeeded so I'm a bit nervous.

**Comment by skliper** [comment]
_Trac comment by jhageman on 2019-05-06 12:49:00:_

Is this ready for code review (clean compile?)

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-05-24 14:02:27:_

I have rebased this to commit [changeset:cdee882] and confirmed this builds with the latest development branch.

**Comment by skliper** [comment]
_Trac comment by jhageman on 2019-06-13 11:29:07:_

CCB 6/12/19 - CCB code reviewed and approved

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-06-25 13:11:04:_

Merged into `ic-ccb-2019-06-12`

**Comment by skliper** [comment]
_Trac comment by jhageman on 2019-07-08 09:48:39:_

Moved 1.3.1 tickets planned for closure in "end-of-summer" 2019 release to 1.4.0

**Comment by skliper** [comment]
_Trac comment by jhageman on 2019-07-10 13:27:49:_

CCB 7/10/2019 - Approved for merge to development branch

**Comment by skliper** [comment]
_Trac comment by jphickey on 2019-07-11 14:34:01:_

Now merged to "development" branch
